examples/acm/raw/info.dat

```diff
@@ -0,0 +1,63 @@
+{
+    "node.dat": {
+        "node type": {
+            "0": "paper",
+            "1": "author",
+            "2": "subject",
+            "3": "term"
+        }
+    },
+    "link.dat": {
+        "link type": {
+            "0": {
+                "start": 0,
+                "end": 0,
+                "meaning": "paper-cite-paper"
+            },
+            "1": {
+                "start": 0,
+                "end": 0,
+                "meaning": "paper-ref-paper"
+            },
+            "2": {
+                "start": 0,
+                "end": 1,
+                "meaning": "paper-author"
+            },
+            "3": {
+                "start": 1,
+                "end": 0,
+                "meaning": "author-paper"
+            },
+            "4": {
+                "start": 0,
+                "end": 2,
+                "meaning": "paper-subject"
+            },
+            "5": {
+                "start": 2,
+                "end": 0,
+                "meaning": "subject-paper"
+            },
+            "6": {
+                "start": 0,
+                "end": 3,
+                "meaning": "paper-term"
+            },
+            "7": {
+                "start": 3,
+                "end": 0,
+                "meaning": "term-paper"
+            }
+        }
+    },
+    "label.dat": {
+        "node type": {
+            "0": {
+                "0": "database",
+                "1": "wireless communication",
+                "2": "data mining"
+            }
+        }
+    }
+}
```

Comment:
Remember to delete these data files. Users will use your dataset to download these data.